official/resnet/README.md

```diff
@@ -80,7 +80,22 @@ The appropriate distribution strategy is chosen based on the `--num_gpus` flag.
 By default this flag is one if TensorFlow is compiled with CUDA, and zero
```

Comment:
You should feel free to re-organize this whole section to fit your needs. the current organization seems a bit odd to me, and we don't necessarily need to shoe-horn into it.